media-ranker/app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,50 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.order(rank: :desc)
+  end
+
+  def create
+    album = Album.create(album_params)
+    album.update_attribute(:rank, 1)
+    redirect_to albums_path
+  end
+
+  def new
+    @album = Album.new
+  end
+
+  def edit
+    @album = Album.find(params[:id])
```

Comment:
Since basically all these controllers' methods require `@media = Media.find(params[:id])`, using a `before_action` in each controller could clean things up a little.